dissect/target/plugins/filesystem/walkfs.py

```diff
@@ -41,8 +51,43 @@ def check_compatible(self) -> None:
 
     @export(record=FilesystemRecord)
     @arg("--walkfs-path", default="/", help="path to recursively walk")
-    def walkfs(self, walkfs_path: str = "/") -> Iterator[FilesystemRecord]:
-        """Walk a target's filesystem and return all filesystem entries."""
+    @arg("--suid", action="store_true", help="output suid records")
+    @arg("--capability", action="store_true", help="output capability records")
+    def walkfs(self, walkfs_path: str = "/", suid: bool = False, capability: bool = False) -> Iterator[WalkFsRecord]:
+        """Walk a target's filesystem and return all filesystem entries.
+
+        Can return additional SUID and Capability records when given ``--suid`` and/or ``--capability`` arguments.
+
+        XFS and ExtFS filesystems can set extended attributes (``xattr``) on filesystem entries.
```

Comment:
Other filesystems can too, but I think we only implement it somewhat okay-ish for these at the moment? Maybe refine this sentence a bit.